_pages/en/software-assessment-sharing-pilot.md

```diff
@@ -0,0 +1,136 @@
+- **Users’ Security and Productivity** - Bugs/issues/workarounds and security vulnerabilities are only known by the department(s) who discovered it
+
+## Expected Results
+
+The pilot aims to reduce the workload required for software assessments across the GC by providing departments access to software assessment documents performed by other departments for information purposes as a starting point to their own processes.
+It will also bring together the software assessment community in the GC and increase collaboration between departments.
+Other impacts include:
+
+- Less duplication and costs 
+- Assessors can dedicate their time to high-value work 
+- Higher quality of assessments 
+- Higher employee satisfaction 
+- Better security and productivity for end users 
+- Reduced risks (by awareness of other departments using the same application) 
+- Detailed audit trail (version control)
+- Greater public value from data
+- Enhanced evidence-informed decision-making
+
+### Align with Canada’s Digital Ambition
+
+[Canada’s Digital Ambition 2022](https://www.canada.ca/en/government/system/digital-government/government-canada-digital-operations-strategic-plans/canada-digital-ambition.html#toc4) defines digital Government as:
+
+> The GC is an open and service-oriented organization that operates and delivers programs and services to people and businesses in simple, modern and effective ways that are optimized for digital and available anytime, anywhere and from any device.
+> Digitally, the GC must **operate as one** to benefit the people of Canada.
+
+The GC is already one organization in the legal sense.
+Sharing documents created in one department for the benefit of the others should be the norm.
+
+This direction is also reflected in the Directive on Open Government (Maximizing the release of GC information) and the GC Digital Standards (Work in the open by default and Collaborate widely).
+
+## GCxchange Group
+
+GCxchange is an instance of Sharepoint/Teams making a good platform to create a shared document library and space to dicsuss software assessments in the GC.
+The GCexchange group is accessible to most of GC departments already.
+GC Employees can see what documents have been shared and participate in discussions about software assessments.
+
+Note: **Only Unclassified information can be shared on GCxchange** and that is what we want to share as part of the pilot.
+
+## Sharing ESDC Software Assessments
+
+Before software can be used at ESDC, it must be assessed for architecture, accessibility, security and privacy.
+The teams responsible for software assessments at ESDC agreed to share templates and the assessments they produce during the software intake process.
+There were legitimate concerns about what we can and can't share and if these are really protected or classified documents.
```

Comment:
that last sentence should be reworded.
More about "the concerns regarding sensitivity of some software assessments are being worked on with the respective teams in order to ensure only unclassified documents are being shared on GCExchange"